<launch>
    <include file="$(find bot_moveit)/launch/move_group.launch"/>
    <node name="basic_grasping_perception" pkg="simple_grasping" type="basic_grasping_perception" >
        <rosparam command="load" file="$(find bot_gazebo)/config/simple_grasping.yaml" />
    </node>
        <!--<node name="vision_server" pkg="bot_gazebo" type="vision_server.py" />-->

</launch>